go.mod

```diff
@@ -3,17 +3,16 @@ module github.com/CrunchyData/crunchy-bridge-operator
 go 1.16
 
 require (
-	github.com/RHEcosystemAppEng/dbaas-operator v0.1.4-0.20220601181250-cc7bd85ce9c0
-	github.com/go-logr/logr v0.4.0
-	github.com/google/uuid v1.1.2
+	github.com/RHEcosystemAppEng/dbaas-operator v1.0.1-0.20220825183828-b5d7b8e52699
```

Comment:
@jeremyary @tchughesiv just as note we can update this release version of dbaas operator, when we publish next release.